### ISSUE TITLE
Confirm tests and specs align

### ISSUE THREAD

**Issue by marcoscaceres** [w3c/browser-payment-api#779]
We need to make sure that the tests listed in the spec and the actual tests in the test suite align. 

We need:

  1. The list of files in WPT's [payment request](https://github.com/web-platform-tests/wpt/tree/master/payment-request) directory.
  1. To be compared to the "data-test" files listed in [the spec](https://w3c.github.io/payment-request/) ([example](https://github.com/w3c/payment-request/blob/gh-pages/index.html#L4898)). 

Then we can figure out: 

 * Are all tests in the test suite in the spec?
 * Are tests listed in the spec that don't actually link to anything?

@Sylvia23 has offered to help us with this 👋

The tests that are in the payment request directory but not in the spec are: 

- [x] MerchantValidationEvent/complete-method.https.html
- [ ] PaymentAddress/attributes-and-toJSON-method-manual.https.html
- [ ] PaymentMethodChangeEvent/methodDetails-attribute.https.html
- [ ] PaymentMethodChangeEvent/methodName-attribute.https.html
- [ ] PaymentRequestUpdateEvent/updateWith-call-immediate-manual.https.html
- [ ] PaymentRequestUpdateEvent/updateWith-method-abort-update-manual.https.html
- [ ] PaymentRequestUpdateEvent/updateWith-state-checks-manual.https.html
- [ ] PaymentRequestUpdateEvent/updatewith-method.https.html
- [ ] PaymentValidationErrors/retry-shows-payer-member-manual.https.html
- [ ] PaymentValidationErrors/retry-shows-shippingAddress-member-manual.https.html
- [ ] payment-response/onpayerdetailchange-attribute-manual.https.html
- [ ] payment-response/onpayerdetailchange-attribute.https.html
- [x] payment-request-abort-method.https.html
- [x] payment-request-canmakepayment-method.https.html
- [ ] payment-request-not-exposed.https.worker.js

-The tests that are in the spec but are not present in or are wrongly addressed in the payment directory are:

### Line 640:
- [x] active-document-cross-origin.https.sub.html
- [x] active-document-same-origin.https.html
- [x] removing-allowpaymentrequest.https.sub.html
- [x] setting-allowpaymentrequest-timing.https.sub.html
- [x] setting-allowpaymentrequest.https.sub.html

### Line 904:
- [x] payment-request-show-method-manual.https.html

### Line 910:
- [x] payment-request-show-method-manual.https.html
### Line 1138:
- [x] payment-request-abort-method-manual.https.html
### Line 1197:
- [x] payment-request-canmakepayment-method-manual.https.html
### Line 1300:
- [x] payment-request/onmerchantvalidation-attribute.https.html
### Line 3290:
- [x] payment-request/PaymentValidationErrors/retry-shows-error-member-manual.https.html
### Line 3322:
- [x] payment-request/payment-response/rejects_if_not_active-manual.https.html
### Line 3630:
- [x] payment-request/payment-response/retry-method-manual.https.html
### Line 4177:
- [x] payment-request-update-event-updatewith-method.https.html

**Comment by Sylvia23** [comment]
Thank you so much @marcoscaceres for writing down this issue. I would love to help with this issue and report back in a while! Thanks!

**Comment by Sylvia23** [comment]
@marcoscaceres The part of the issue - "There are tests in the spec but are not present in or are wrongly addressed in the payment directory" has been fixed with #788 . Please review. Thanks!

**Comment by marcoscaceres** [comment]
@Sylvia23, great work so far. Could you let us know what’s left to be done?

**Comment by Sylvia23** [comment]
Thank you so much @marcoscaceres . Now, we are left with adding the extra test suites that are there in the payment request repository but are missing in our spec. So, we need to add those in the spec. I have again checked the test suites allignment after merging #788 . I found that we can now check the boxes for MerchantValidationEvent/complete-method.https.html , payment-request-abort-method.https.html, payment-request-canmakepayment-method.https.html in our checklist because they got resolved automatically by #788 . Rest of the tests still need to be worked upon. I am reading the tests and their attributes to get an idea about what they are about so as to put them in their appropriate position in the spec. Any kind of suggestion is highly welcomed. Thanks!
